index.d.ts

```diff
@@ -0,0 +1,49 @@
+declare module 'atomizer' {
+    export interface AtomizerConfig {
+        custom?: Record<string, string>;
+        breakPoints?: Record<string, string>;
+        classNames?: string[];
+    }
+
+    export interface AtomizerOptions {
+        verbose?: boolean;
+    }
+
+    export interface AtomizerRule {
```

Comment:
Need to distinguish between rules and helpers, helpers have different values, for instance: https://github.com/acss-io/atomizer/blob/master/src/helpers.js#L186-L211

Reply:
I'll just relax the rules as `any` for now and we can enhance later on - it seems like it's a pretty complex object that could be broken down into smaller parts if we want to flesh this out more.

Reply:
Rules and helpers are processed through the same APIs, just have different shape

Reply:
updated https://github.com/acss-io/atomizer/pull/370/commits/a315f653e99b60c80408443f7a0a65b01dc6e987